pynumdiff/total_variation_regularization/_total_variation_regularization.py

```diff
@@ -250,68 +251,16 @@ def jerk_sliding(x, dt, params=None, options=None, gamma=None, solver=None):
         gamma = params[0] if isinstance(params, list) else params
         if options != None:
             if 'solver' in options: solver = options['solver']
+            if 'window_size' in options: window_size = options['window_size']
```

Comment:
I've allowed users to pass this via the old way too, just so I can be consistent in my tests.